packages/frontend/src/main/components/comments/CommentEditor.vue

```diff
@@ -157,8 +167,8 @@ export default Vue.extend({
     },
     placeholder(): string {
       return this.addingComment
-        ? 'Your comment... (press enter to send)'
-        : 'Reply... (press enter to send)'
+        ? 'Your comment... (Enter sends it)'
+        : 'Reply... (Enter sends it)'
```

Comment:
'Press enter to send' is the more commonly used phrase

Reply:
but this hurts my eyes
![image](https://user-images.githubusercontent.com/57442769/181735004-99d13802-e960-4a76-b108-10fca6b9f663.png)

Reply:
I agree that "Press enter to send" is more common. Maybe we can just remove the "Your comment..." prefix? It's kinda self-explanatory that your comment can go there